Tools/PIDML/qaPidML.cxx

```diff
@@ -326,15 +324,9 @@ struct pidml {
   void pidML(const T& track, const int pdgCodeMC)
   {
     float pidCertainties[3];
-    if (track.p() < pSwitchValue[i]) {
```

Comment:
After moving mPTLimits to PidOnnxModel, you could use pSwitchValues as configurables here.